### DISCUSSION TITLE
Naming the Project

### DISCUSSION THREAD

**Discussion by xynydev** [blue-build/.github#1] [Polls]
Since this is a new project, we should choose a clear name.

IMO this name should be related to at least one of:
- Building
- Images
- Recipes (cooking)
- Blue (know your legacy: ublue, Fedora blue)

**Comment by fiftydinar** [comment]
I vote for BlueBuild.

However, which variant of this name is better?

- BlueBuild
- Blue-Build
- Bluebuild

**Reply by fiftydinar** [reply]
BlueBuild still seems best to me

**Reply by xynydev** [reply]
The dash in the org name and possible domain is just because of the original name being taken, it's not good looking or a good "product" name IMO. I like the PascalCase naming, it separates the words better, so that's what I'd use in logos and official writing. It doesn't really matter how it's capitalized in chatting, though, like ublue vs Ublue vs uBlue.

**Reply by gmpinder** [reply]
Yeah I'm a big fan of PascalCase for the name

**Comment by xynydev** [comment]
Closing this, I think the matter is settled.
